special-pages/pages/new-tab/app/mock-transport.js

```diff
@@ -586,7 +586,7 @@ export function initialSetup(url) {
     widgetsFromStorage.push({ id: 'protections' });
     widgetConfigFromStorage.push({ id: 'protections', visibility: 'visible' });
 
-    if (url.searchParams.has('omnibar')) {
+    if (url.searchParams.get('omnibar') !== 'false') {
```

Comment:
Not related but figure it’s time we flip it so that omnibar widget is enabled by default.